README.md

```diff
@@ -70,8 +70,20 @@ npx superstream-kafka-analyzer
 ```bash
 # Using a configuration file
 npx superstream-kafka-analyzer --config config.json
+
+# Using a configuration file and uploading results to NATS
+npx superstream-kafka-analyzer --config config.json --nats-config nats-config.json
 ```
 
+### NATS Config Mode (Upload to NATS)
```

Comment:
This is a hidden option for our use let's not confuse users with it. so let's remove Nats related things where it is customer facing